tools/ci_build/github/windows/extract_nuget_files_gpu.ps1

```diff
@@ -29,7 +29,9 @@ Foreach-Object {
 New-Item -Path $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\_deps\protobuf-build\RelWithDebInfo -ItemType directory
 
 Copy-Item -Path $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo\nuget-artifacts\onnxruntime-win-x64-cuda-*\lib\* -Destination $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo
-Copy-Item -Path $Env:BUILD_BINARIESDIRECTORY\extra-artifact\protoc.exe $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\_deps\protobuf-build\RelWithDebInfo
+dotnet new console
+dotnet add package Google.Protobuf.Tools --version 3.21.12
+Copy-Item -Path $Env:USERPROFILE\.nuget\packages\google.protobuf.tools\**\tools\windows_x64\protoc.exe $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\_deps\protobuf-build\RelWithDebInfo
```

Comment:
If it is in USERPROFILE folder, the file could come from a different run. Can we put it in the current workspace?

Reply:
put it into $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\_deps\protobuf-build